data/migrations/README.md

```diff
@@ -0,0 +1,16 @@
+# Migrations for sovrin-node
+
+All migrations are python scripts with names satisfying the following rule:
+```
+<underscored_version-migration-should-be-applied-to>_<migration-name>
```

Comment:
I am feeling concerned about two issues with this approach:
1. This creates multiple migration scripts per version change. I want one migration script per update. Anything more is unnecessary complexity. Also, it gives multiple points of failure instead of one, and it gives no responsibility to these individual scripts to know about or coordinate with one another. I think it increases the chances of failure.
2. This mechanism doesn't include in its name the version you are coming from, only the version you are headed to. I think this is problematic because we might have nodes upgrading from more than one starting point.